.eslintrc

```diff
@@ -0,0 +1,4 @@
+{
+    "extends": ["@imaginary-cloud/react"]
+
```

Comment:
**setup eslint with AirBnB  as popular style guide**
> - https://www.andrewmin.info/blog/react-setup/
**add rules to configure linebreak-style**
> - ![eslint-errors](https://user-images.githubusercontent.com/50295185/105055538-d838a400-5a7b-11eb-9b05-faaa3a58ecef.PNG)

Reply:
Eslint is already configured, thank you